viewer/packages/tools/src/Toolbar/ToolbarTool.ts

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { CogniteModelBase, Cognite3DViewer } from '@reveal/core';
+import { Cognite3DViewerToolBase } from '../Cognite3DViewerToolBase';
+import { Toolbar, ToolbarPosition } from './Toolbar';
+import { AxisViewTool } from '../AxisView/AxisViewTool';
+import { CameraControlsOptions } from '@reveal/core';
+
+import iconSet12 from './icons/Cognite_Icon_Set-12.png';
+import iconSet14 from './icons/Cognite_Icon_Set-14.png';
+import iconSet19 from './icons/Cognite_Icon_Set-19.png';
+import iconSet30 from './icons/Cognite_Icon_Set-30.png';
+import iconSet32 from './icons/Cognite_Icon_Set-32.png';
+
+/**
+ * Tool to help user to use the default toolbar items such as Camera & other features in Reveal to enable/disable it.
+ * The tool also helps in adding or create custom toolbar.
+ */
+export class ToolbarTool extends Cognite3DViewerToolBase {
+  private readonly _viewer: Cognite3DViewer;
+  private readonly _model: CogniteModelBase;
+  private readonly _toolbar: Toolbar;
+  private axisTool: AxisViewTool | undefined;
+  private cameraControlOption: CameraControlsOptions;
+
+  constructor(viewer: Cognite3DViewer, model: CogniteModelBase) {
```

Comment:
Avoid limiting `ToolbarTool` to only work on a single model. I think the only function that requires model is `fitCameraToModel`. I think we instead should add a `Cognite3dViewer.fitCameraToAllModels` and use this from the tool.

Reply:
`Cognite3dViewer.fitCameraToAllModels` and use it from the tool make sense, will add and check the result